ola.spec

```diff
@@ -1,91 +1,109 @@
-%define name    ola
-%define version 0.3.1
-%define release %mkrel 1
-
-
-Name:      %{name}
-Version:   %{version}
-Release:   %{release}
-Summary:   OLA - Open Lighting Architecture
-Group:     Other
-License:   GPL
-URL:       https://github.com/OpenLightingProject/ola
-# Other doc found at:
-#  http://www.opendmx.net/index.php/LLA_0.3
-#  http://www.opendmx.net/index.php/LLA_on_Linux
-Source0:   %{name}/%{name}-%{version}.tar.gz
-BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root
-BuildRequires: libmicrohttpd-devel >= 0.4.0, libcppunit-devel, protobuf-devel >= 2.1.0
-Requires:      libmicrohttpd >= 0.4.0, libcppunit, protobuf >= 2.1.0
+Name:           ola
+Version:        0.9.7
+Release:        1
+Summary:        Open Lighting Architecture
+Group:          Applications/Multimedia
+License:        GPLv2 and LGPLv2
+URL:            https://github.com/OpenLightingProject/ola
+Source:         https://github.com/OpenLightingProject/ola/releases/download/%{version}/%{name}-%{version}.tar.gz
+BuildRoot:      %{_buildrootdir}/%{name}-%{version}-%{release}-root
+BuildRequires:  libmicrohttpd-devel,cppunit-devel,protobuf-devel,protobuf-compiler,protobuf-python,libftdi-devel,openslp-devel,uuid-devel,libtool,bison,flex,pkgconfig,gcc,gcc-c++,python-devel,avahi-devel
 
 %description
-The Open Lighting Architecture (OLA) consists of two parts, the daemon olad and the library, libola.
-
-
-%package -n libola
-Group:         Other
-Summary:       OLA - Open Lighting Architecture
-
-%description -n libola
-The OLA library
-
-
-%package -n libola-devel
-Group:         Other
-Summary:       OLA - Open Lighting Architecture
-
-%description -n libola-devel
-The OLA library headers
-
+The Open Lighting Architecture is a framework for lighting control information.
+It supports a range of protocols and over a dozen USB devices. It can run as a
+standalone service, which is useful for converting signals between protocols,
+or alternatively using the OLA API, it can be used as the back-end for lighting
+control software. OLA runs on many different platforms including ARM, which
+makes it a perfect fit for low cost Ethernet to DMX gateways.
+
+%package devel
+Requires:      ola = %{version}-%{release}, protobuf-devel
+Group:         Development/Libraries
+Summary:       C/C++ Development files for OLA
+
+%description devel
+The OLA C/C++ library
+
+%package -n python2-%{name}
+Requires:      ola = %{version}-%{release}, protobuf-python, python(abi) = 2.7
```

Comment:
2.6 seems to be our minimum, but you want abi >= 2.6, rather than less than or equal to 2.7, as 2.5 won't work.  You also need to restrict it to less than 3.